go.mod

```
@@ -1,4 +1,4 @@
-module github.com/aquasecurity/esquery
+module github.com/hardy4yooz/esquery
```

Please change to aquasecurity 
```suggestion
module github.com/aquasecurity/esquery
```

---

yes,that is.